web-ng/src/app/components/project-header/project-header.component.ts

```diff
@@ -94,6 +97,15 @@ export class ProjectHeaderComponent implements OnInit, OnDestroy {
     });
   }
 
+  initPermission(): void {
+    const project = this.projectService.getCurrentProject();
+    if (!project) {
+      return;
+    }
+    const acl = this.projectService.getProjectAcl(project);
+    this.canShare = this.authService.canManageProject(acl);
```

Comment:
We don't need to cache `canShare` as an internal state; just add a method `canShare()` that calls the auth service on the ACL. Then we won't need `initPermissions` either.